src/components/SocialButtons.tsx

```diff
@@ -0,0 +1,41 @@
+import Image from "next/image";
+
+export default function SocialButtons() {
```

Comment:
Change to const SocialButtons = () => { and then export default at the bottom just to keep a convention